skills/codex-review-partner/scripts/review_supervisor.py

```
@@ -0,0 +1,534 @@
+    if stop_reason:
+        cleanup_private_session(adapter, leader)
+        raise SupervisorError("review supervisor received a stop signal before Codex launch")
+    if not identity_matches(adapter, arguments.parent_pid, arguments.parent_start_identity, arguments.parent_boot_id):
+        raise SupervisorError("review launcher parent exited before Codex launch")
+    if not identity_matches(adapter, arguments.owner_pid, arguments.owner_start_identity, arguments.owner_boot_id):
+        raise SupervisorError("Pi owner exited before Codex launch")
+
+    environment = os.environ.copy()
+    environment["CODEX_REVIEW_PARTNER_ACTIVE"] = "1"
+    os.chdir(arguments.work_dir)
+    child = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)